### PR TITLE
Merge VisualStudioSetupInteractive into VisualStudioInteractiveComponents

### PR DESCRIPTION
**Pull request by tmat**
Fixes setup authoring of Roslyn interactive components.

Currently interactive components are installed as two VSIX extensions, each ends up installed in a different directory in Willow and in F5 scenario (in Dev14 we merged everything into PrivateAssemblies in setup authoring). One directory contains .rsp file and *.Repl.dll and the other the interactive host and its dependencies. Such layout prevents the .rsp file to refer to a dll collocated with the interactive host since it doesn't know the relative path between these two. The change removes the unnecessary VSIX and merges it to VisualStudioInteractiveComponents.

### CODE REVIEW

**Comment by tmat** [comment]
@jasonmalinowski @KevinH-MS @brettfo 
@dotnet/roslyn-interactive

**Comment by brettfo** [comment]
:shipit: I'm all for condensing binaries/packages.

File: docs/contributing/Building, Debugging, and Testing on Windows.md [hunk 1]
```diff
@@ -83,18 +83,6 @@ deployed.
...
-  Interactive folder from the Solution Explorer, and builds 
-  Microsoft.VisualStudio.VsInteractiveWindow.vsix.
-  This includes the "base" interactive window experience that is shared by
-  Roslyn, Python, and other languages. This code is core support only and
-  doesn't include any language-specific logic.
-- **VisualStudioSetupInteractive**: this project can be found inside the
-  Interactive\Setup folder from the Solution Explorer, and builds
-  Roslyn.VisualStudio.Setup.Interactive.vsix.
-  It deploys the Roslyn (i.e. C# and VB) specific parts of the interactive
-  window. If you're working on the interactive experience, this is the project
```
Comment (jasonmalinowski) [inline docs/contributing/Building, Debugging, and Testing on Windows.md]: Should this text be placed somewhere else now about what to F5 for interactive stuff?

**Comment by jasonmalinowski** [comment]
@dotnet-bot test vsi please.

**Comment by tmat** [comment]
Integration tests need another change in closed: https://github.com/dotnet/roslyn-internal/pull/1313

**Comment by tmat** [comment]
Re F5: works on my machine against Dev15 pre-Willow

**Comment by tmat** [comment]
@MattGertz Ask mode PR. Interactive Window is broken in the current lab/ml build.

**Comment by MattGertz** [comment]
Approved pending the usual.

**Comment by tmat** [comment]
@dotnet-bot test vsi please.

**Pull request merged**